pkg/defaults/defaults.go

```diff
@@ -576,4 +576,7 @@ var (
 		"cilium_lb6_source_range":   "enabled,128,0",
 		"cilium_lb6_affinity_match": "enabled,128,0",
 	}
+
+	// ClusterHealthAddrs is the default value for option.ClusterHealthAddrs
+	ClusterHealthAddrs = []string{"0.0.0.0"}
```

Comment:
Will this not bind only to IPv4? Shouldn't we use an empty string here?

Reply:
I guess you are correct, upstream issue: https://github.com/golang/go/issues/48723

What was the Viper issue you were experiencing? I'd expect `[]string{""}` to work here (though Viper is known to have bugs)

Reply:
Ok, my bad. It's not related to Viper but Cobra. Cobra interprets `[]string{""}` the same as `[]string{}` within an default value of `flags.StringSlice()` function:
```
package main

import (
	"fmt"

	"github.com/spf13/cobra"
)

func main() {
	var RootCmd = &cobra.Command{}
	flags := RootCmd.Flags()
	flags.StringSlice("string-slice-test", []string{""}, "...")
	s, _ := flags.GetStringSlice("string-slice-test")
	fmt.Printf("s: %v", s)
}
```
Output:
```
s: []
Program exited.
```

Reply:
Ok, my bad. It's not related to Viper but Cobra. Cobra interprets []string{""} the same as []string{} within an default value of flags.StringSlice() function:
```
package main

import (
	"fmt"

	"github.com/spf13/cobra"
)

func main() {
	var RootCmd = &cobra.Command{}
	flags := RootCmd.Flags()
	flags.StringSlice("string-slice-test", []string{""}, "...")
	s, _ := flags.GetStringSlice("string-slice-test")
	fmt.Printf("length of s: %v", len(s))
}
```
Output:
```
length of s: 0
Program exited.
```

Reply:
Wow :exploding_head: Thanks for clarifying